### PR TITLE
Support for nulls on primitives when parsing TDS

### PR DESCRIPTION
**Pull request by rafaelbey**
Allow primitive types to be null. For int/long, using largest primes as unlikely we would see these on our test cases, rather than using min int/long, as these can be used for testing boundary conditions.

### CODE REVIEW

File: legend-pure-dsl/legend-pure-dsl-tds/legend-pure-m2-dsl-tds-grammar/src/main/java/org/finos/legend/pure/m2/inlinedsl/tds/TDSExtension.java [hunk 1]
```diff
@@ -236,17 +237,22 @@ private static String convertType(DataType dataType)
         }
     }
 
-    private static SinkFactory makeMySinkFactory()
+    public static CsvSpecs makePureCsvSpecs()
```
Comment (rafaelbey) [inline legend-pure-dsl/legend-pure-dsl-tds/legend-pure-m2-dsl-tds-grammar/src/main/java/org/finos/legend/pure/m2/inlinedsl/tds/TDSExtension.java:240]: Made this utility methods public to allow using them on Legend Engine

File: legend-pure-dsl/legend-pure-dsl-tds/legend-pure-m2-dsl-tds-grammar/src/main/java/org/finos/legend/pure/m2/inlinedsl/tds/TDSExtension.java [hunk 2]
```diff
@@ -236,17 +237,22 @@ private static String convertType(DataType dataType)
...
+    {
+        return CsvSpecs.builder().nullValueLiterals(Arrays.asList("", "null")).build();
```
Comment (rafaelbey) [inline legend-pure-dsl/legend-pure-dsl-tds/legend-pure-m2-dsl-tds-grammar/src/main/java/org/finos/legend/pure/m2/inlinedsl/tds/TDSExtension.java:242]: When parsing, either empty or the literal null will be treated as null values.  This is mimicking what we already do on Engine.

**Review by pierredebelen — APPROVED** [review]
